src/config/graphical-session/xorg.md

```diff
@@ -24,9 +24,13 @@ Void provides both open-source and proprietary (non-free) video drivers.
 
 ### Open Source Drivers
 
-The open source drivers are installed with the `xorg` package by default, or may
-be installed individually if the `xorg-minimal` was installed. Below is a table
-of device brands and their driver packages.
+Xorg can use two categories of open source drivers: DDX or modesetting.
+
+#### DDX
+
+The DDX drivers are installed with the `xorg` package by default, or may be
+installed individually if the `xorg-minimal` was installed. Below is a table of
+device brands and their driver packages.
 
 | Brand  | Type        | Driver Package       |
```

Comment:
I don't like this table, it's inaccurate (`xf86-video-ati` is not only for old ATi GPUs) and incomplete (we have a lot more drivers than that)

Reply:
Do you think we should try to document all of them instead? Or perhaps find some upstream docs with all of those drivers which we can link to.

Reply:
Indeed, the table is not very useful. I wouldn't list everything, I'd just say that the DDX are provided by the `xf86-video-*` packages. The only non-obvious choice is `xf86-video-ati` vs `xf86-video-amdgpu`, which would be better documented on a future AMD subsection of Graphics Drivers.

Reply:
Table removed